packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -66,12 +66,13 @@ class DDSTabsExtended extends StableSelectorMixin(LitElement) {
     this._tabItems.map((tab, index) => {
       (tab as DDSTab).selected = index === this._activeTab;
       (tab as DDSTab).setIndex(index);
-      const navLink = this.shadowRoot!.querySelectorAll('.bx--tabs__nav-link div p')[index];
-      if (navLink.scrollHeight > 70) {
+      const navLink = this.shadowRoot!.querySelectorAll('.bx--tabs__nav-link')[index];
```

Comment:
Looks great!! Awesome first PR! one small change: 
```suggestion
      const navLink = this.shadowRoot!.querySelectorAll(`.${prefix}--tabs__nav-link`)[index];
```